website/docs/cdktf/typescript/r/lb_target_group.html.markdown

```diff
@@ -162,7 +162,8 @@ This resource supports the following arguments:
 * `deregistrationDelay` - (Optional) Amount time for Elastic Load Balancing to wait before changing the state of a deregistering target from draining to unused. The range is 0-3600 seconds. The default value is 300 seconds.
 * `healthCheck` - (Optional, Maximum of 1) Health Check configuration block. Detailed below.
 * `lambdaMultiValueHeadersEnabled` - (Optional) Whether the request and response headers exchanged between the load balancer and the Lambda function include arrays of values or strings. Only applies when `target_type` is `lambda`. Default is `false`.
-* `loadBalancingAlgorithmType` - (Optional) Determines how the load balancer selects targets when routing requests. Only applicable for Application Load Balancer Target Groups. The value is `round_robin` or `least_outstanding_requests`. The default is `round_robin`.
+* `loadBalancingAlgorithmType` - (Optional) Determines how the load balancer selects targets when routing requests. Only applicable for Application Load Balancer Target Groups. The value is `round_robin`, `least_outstanding_requests`, or `weighted_random`. The default is `round_robin`.
+* `loadBalancingAnomalyMitigation` - (Optional) Determines whether to enable target anomaly mitigation.  Target anomaly mitigation is only supported by the `weighted_random` load balancing algorithm type.  See [doc](https://docs.aws.amazon.com/elasticloadbalancing/latest/application/load-balancer-target-groups.html#automatic-target-weights) for more information.  The value is `"on"` or `"off"`. The default is `"off"`.
```

Comment:
I'm not too familiar with CDK but I'm assuming this conversion was correct?

Reply:
Not necessary as the CDKTF docs are auto-generated (we are updating the Contributor Guide to reflect this).